released/packages/coq-mathcomp-analysis/coq-mathcomp-analysis.0.1.0/opam

```diff
@@ -18,23 +18,24 @@ build: [
 install: [
   [make "install"]
 ]
-remove: ["rm" "-R" "%{lib}%/coq/user-contrib/mathcomp/analysis"]
 depends: [
-  "coq" { ((>= "8.7" & < "8.9~") | = "dev") }
+  "coq" { ((>= "8.8" & < "8.9~") | = "dev") }
```

Comment:
```suggestion
  "coq" { (>= "8.8" & < "8.9~") }
```
Can we remove the `dev` option here as well? I highly doubt that this works on current Coq master, and I think it's bad practice to allow `dev` for Coq in packages in `released`. Thumbs up for other changes.

Reply:
Thanks